cluster/sync.sh

```diff
@@ -35,9 +35,16 @@ done
 
 ./cluster/kubectl.sh apply -f ./config/test/kubemacpool.yaml
 
-# Make sure all containers are ready
-while [ -n "$(./cluster/kubectl.sh get pods --all-namespaces -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers | grep false)" ]; do
-    echo "Waiting for all containers to become ready ..."
-    ./cluster/kubectl.sh get pods --all-namespaces -o'custom-columns=status:status.containerStatuses[*].ready,metadata:metadata.name' --no-headers
-    sleep 10
-done
+pods_ready_wait() {
+  echo "Waiting for non-kubemacpool containers to be ready ..."
+  ./cluster/kubectl.sh wait pod --all -n kube-system --for=condition=Ready --timeout=5m
+
+  echo "Waiting for (TBD) to be ready ..."
+#  cluster/kubectl.sh wait --for=condition=available -f ./config/test/kubemacpool.yaml --timeout=3m
+#  cluster/kubectl.sh wait -n kubemacpool-system --for=condition=available deployment.apps/kubemacpool-mac-controller-manager --timeout=2m
```

Comment:
I did not manage to find a proper kubectl wait to be satisfied when the leader is ready.
the issue is that during make cluster-sync `./cluster/kubectl.sh wait pod -n kubemacpool-system --for=condition=Ready -l kubemacpool-leader=true --timeout=2m` receives error: no matching resources found.

if I sleep 5s then the former wait waits and eventually succeeds. 
Do you have any suggestions?